### ISSUE TITLE
Problem of touch issues.

### ISSUE THREAD

**Issue by johnlu6518** [commaai/openpilot#20841]
The options for reboot and turn off are difficult to click on the screen. 
(The touch is not smooth, and the display cannot be moved up and down)

 Device:comma two
 Dongle ID: ec30903bb15db3ad

**Comment by adeebshihadeh** [comment]
Can you be more specific? It's the normal android power menu, which doesn't move.

**Comment by johnlu6518** [comment]
Ok，I will take photos or video for u later.

**Comment by johnlu6518** [comment]
https://youtube.com/shorts/f_BMZKt-2iU?feature=share

**Comment by VirtuallyChris** [comment]
From the video it looks like you tried to scroll from the bottom of the screen up; but you can only scroll within the box with the actual settings in it. If you scroll with your finger in the middle of the screen to start, are there any issues?

**Comment by johnlu6518** [comment]
Yes, you are right. Scroll with finger in the middle of the screen.
